jbot/src/main/java/me/ramswaroop/jbot/core/facebook/Bot.java

```diff
@@ -265,7 +265,7 @@ private void invokeMethods(Event event) {
 
             methodWrappers = new ArrayList<>(methodWrappers);
             MethodWrapper matchedMethod =
-                    getMethodWithMatchingPatternAndFilterUnmatchedMethods(event.getPatternFromEventType(), methodWrappers);
+                    getMethodWithMatchingPatternAndFilterUnmatchedMethods(getPatternFromEventType(event), methodWrappers);
```

Comment:
What motivated this change? Is this a better coding practice or something else?

Reply:
Without this change none of the tests pass as Jackson fails (that's the stacktrace I left in the comment).

You would have to spend some time digging into the why of it.